node/src/Cardano/Node/API.hs

```diff
@@ -384,6 +339,9 @@ v1SyncPercentage nodeHeight walletHeight =
 -- take up to 30s to resolve.
 --
 -- Copied from Cardano.Wallet.Kernel.NodeStateAdapter
+--
+-- Could potentially throw "blocked indefinitely" but that'd show up as
+-- a timeout to the client.
```

Comment:
Hmmm, this shouldn't _in theory_; Since the MVar is "guaranteed" to be resolved to either `NtpDrift` or `NtpSyncUnavailable` within 30 sec (guaranteed by the networking team). Yet, generally speaking, I agree ^^"

Reply:
When I say "potentially" I mean "as far as a legal interpretation of the types are concerned" :) I trust networking team but the code may drift and guarantees that aren't typed aren't checked.

Reply:
got you.